### ISSUE TITLE
Suggestion - Key Copies

### ISSUE THREAD

**Issue by HollaYalla** [InZidiuZ/op-framework-issue-tracker#3625]
### Assurance

I have read the `README.md`

### Summary

With the current implementation of the Grab Keys System it opens the door for a way to make duplicate keys for your friends maybe just 1 or two where you will go to a store and then buy a Blank Keyfob which is $100 - $250 and when you are inside the vehicle you can use this keyfob to get keys to the car and it will eat the key then you can still thing grab_keys to remove that key from the ignition this will stop the need for everyone to keep lockpicking cars to get keys which is something PD to quite often. If you forget to grab the key from the ignition it will just be gone forever. Most people 100% will forget to grab their keys from the ignition and have to go buy a new one this will remove a lot of money from the economy because everyone is gonna want keys.

Another way to do this is have the owner grab their keys from the iginiton and walk to a place then copy the key but only allow this to be done once or twice so only 2-3 of that key exists at a time

*This idea was helped by Northbayjoe

### Reason

Allow you to give your friends keys and they can actually keep them and remove money from the economy

**Comment by coalaura** [comment]
I already answered this in the opfw discord. The idea behind the key item is the ability to **temporarily** convert the "virtual" key into a "physical" key. There should only ever be 1 key for each vehicle which is why the keys no longer work if you store and withdraw the car again. The keys are there so you don't have to stand next to the car and the person you are wanting to give the keys to. You can leave them in the glovebox, give them to someone, etc.

TL;DR: No
